### PR TITLE
Newsweeks Clone - Milestone 1

### PR DESCRIPTION
**Pull request by RyelBanfield**
A clone of newsweek.com using bootstrap and very little CSS.

### CODE REVIEW

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Approved with OBSERVATIONS :recycle: 

## :wave: :wave: Hello @RyelBanfield and @ar[LINK](https://getbootstrap.com/docs/5.0/utilities/display/) ikarim. Thanks for submitting you project!

There are some issues that need to be addressed before we can move forward. 

While the milestone is approved, it would stop you from advancing in the program if the changes are not made. Before moving to milestone 2 please review and fix the list below.

## Required changes :wrench: 
### [Mandatory] :red_circle: 

- [x] Commit messages should be short but descriptive. use present-tense and don't end the commit message with `.` if you think that the space is too little to explain all the changes send a commit message without using the `git commit -m "some message"` This will open your text editor Example:
```sh
# BAD
Added footer
Fixed z-index property
Update README

#GOOD
Add footer feature
Fix navbar z-index 
Update README's author links
```
- [ ] PR should be descriptive, it should contain a summary of the features and changes done to the project worth mentioning.
- [ ] Branch name should be descriptive, and meaningful. Edit the branch name and update the PR with the new branch. **DO NOT** close the PR and open a new one, just edit and update de PR request. **DO NOT** close the PR. Example: **BAD** `Features` **GOOD** `newsweek-clone`
- [x] The screenshot preview is not displaying any image in the README.
- [x] MIT license link is broken in the README.
- [x] There are no instructions on how to set up and running a local version of the project.
- [x] The Live demo link is broken in the README.
- [ ] Use bootstrap classes to minimize the custom CSS stylesheet. See inline comments. Just give some examples but review the project's CSS style sheet to minimize the use of custom CSS rules.
- [ ] Use efficient CSS selectors. See inline comments.
- [ ] Remove commented code from `index.html`
- [ ] Navbar elements layout is not the same as the original design
  - [ ] date should be assign to the bottom and should match the bottom of the Newsweek logo.
  - [ ] menu elements are not taking the whole width space
  - [ ] the search element should be a form with a search field.
  - [ ] Navbar is not responsive and is not displaying as original in different screen sizes
  image
image
image
image
- [ ] Nav bar elements should change size accordingly to the screen size desktop, tablet, and mobile
- [ ] Footer doesn't look like the original design on table and mobile view.
image
image
image
image
- [ ] Use semantic HTML for `<header>`, and `<nav>` elements. 


## NOTE:
Milestone will appear as **APPROVED** in your Microverse Progress Dashboard. There is no need to submit MIlestones a _re-review_ (This only applies for Milestone 1 and 2). Fix all observations BEFORE moving to the next Milestone. The approval of the overall project will depend on fixing all observations.

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

**Comment by arikarim** [comment]
we fixed all the problems related to the navbar and footer and to use git flow we created new branches then pushed to our feature branch

File: style.css [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+  height: 3000px;
+}
+
+.container-fluid {
+  max-width: 1310px;
+  overflow-x: hidden;
+}
+
+.icon-n {
+  display: block;
```
Comment (Spayco) [inline style.css:12]: `d-block`

File: style.css [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+}
+
+.subscribe-btn {
+  width: 200px;
+}
+
+a {
+  font-size: 16px;
+  padding-right: 10px;
+  white-space: nowrap;
```
Comment (Spayco) [inline style.css:30]: `text-nowrap`

File: style.css [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+  fill: white;
+}
+
+.subscribe-btn {
+  width: 200px;
+}
+
+a {
+  font-size: 16px;
+  padding-right: 10px;
```
Comment (Spayco) [inline style.css:29]: - [ ] Learn how you can use padding on Bootstrap [here](https://getbootstrap.com/docs/4.0/utilities/spacing/)

File: style.css [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+
+/* MEDIA QUERIES */
+
+@media screen and (max-width: 575.98px) {
+  .subscribe-btn,
+  .with-small a {
+    width: 80px;
+    font-size: 0.7rem;
+    padding: 0;
+    margin: 0;
```
Comment (Spayco) [inline style.css:83]: - [ ] Learn how you can use padding/margin on Bootstrap [here](https://getbootstrap.com/docs/4.0/utilities/spacing/)

File: style.css [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+  height: 2px;
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 50%;
+  top: calc(50% - 2px);
+}
+
+.c-vs {
+  position: relative;
```
Comment (Spayco) [inline style.css:117]: `position-relative`

File: style.css [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+  z-index: 1;
+  width: 32px;
+  height: 32px;
+  background-color: #000;
+  color: #d6d6d6;
+  border-radius: 50%;
+  font: bold 14px/32px "Roboto Condensed", sans-serif;
+  text-align: center;
+  margin: 6px auto;
```
Comment (Spayco) [inline style.css:126]: - [ ] Learn how you can use padding/margin on Bootstrap [here](https://getbootstrap.com/docs/4.0/utilities/spacing/)

File: style.css [hunk 7]
```diff
@@ -0,0 +1,132 @@
...
+}
+
+.c-vs {
+  position: relative;
+  z-index: 1;
+  width: 32px;
+  height: 32px;
+  background-color: #000;
+  color: #d6d6d6;
+  border-radius: 50%;
```
Comment (Spayco) [inline style.css:123]: `rounded-circle`

File: style.css [hunk 8]
```diff
@@ -0,0 +1,132 @@
...
+  font: bold 14px/32px "Roboto Condensed", sans-serif;
+  text-align: center;
```
Comment (Spayco) [inline style.css:125]: `text-center`

**Review by Spayco — APPROVED** [review]
# Milestone Approved ✔️ 

Milestone approved, however, there are things that you need to change, you can either make the changes now, or you keep them to the last milestone.

- [x] The content order and disposition changes to the correct place in tablet and mobile
- [x] Colors and font-weight can be improved
- [x] I'll list Bootstrap class names below 👇 in your CSS file, using bootstrap classes whenever possible is required in this project, I might miss some, so double-check :)

### please don't forget to merge to master, good luck with the next projects!


Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Review by Spayco — CHANGES_REQUESTED** [review]
# Changes Requested 

Hello! 👋 
Awesome job so far!! 👍 you still need to make some changes though! 

## Required Changes ♻️ 

- [ ] Check below 👇 

### please make the appropriate changes and ask for another code review ✔️ 

Happy Coding!

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Comment by arikarim** [comment]
Great thanks for the review, we have done the changes and replaced the CSS with bootstrap classes please kindly make another review

**Comment by uryelah** [comment]
Hi **Ryel Banfield** & **Ari Ali Hars**, I'm Sarah your reviewer for this PR 👩‍🚀 

Good job fulfilling the previous TSE requirement!

Your did a great job with the responsiveness of the project overall!

Good luck with your next HTML&CSS projects! 🚀  

-------

## **Your project is approved! 🎊 🥅 💛**

**You may close this Pull Request and merge the changes**

---------

### Optional suggestions

There are a couple of possible improvements I'm suggesting as optional since they were not mentioned in the previous reviews:

- [ ] [OPTIONAL] It's great to use Bootstrap classes, but in this case the Bootstrap red is in a very different shade to the action Newsweek red. Make sure the text, buttons, and backgrounds in red use the correct shade of red.

- [ ] Double check also the color of the navbar yellow button, try to use the same shade the original website uses.

- [ ] [OPTIONAL] Each article image should be a link the same way that the article titles are. Add a link around each image, that's a very common way most webapps behave now days, most images are usually some kind of clickable/link element.

- [ ] [OPTIONAL] Let's add the grey separators between each navbar and footer item.
That's how they original ones look, try to make yours more similar:
header_desktop
footer_desktop

- [ ] [OPTIONAL] Notice how in the original page in desktop size, check the screenshots above, the body background color is a shade of grey visible at the left and right side of the page.
Add that bg color to the body of the page, be careful so it only shows up on wide screens.

---------

    
    
**Keep being an awesome dev and good luck with your next project!**

---------

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Pull request merged**